srcpkgs/taisei/template

```diff
@@ -13,6 +13,8 @@ license="MIT"
 homepage="https://taisei-project.org"
 distfiles="https://github.com/taisei-project/taisei/releases/download/v${version}/taisei-v${version}.tar.xz"
 checksum=dbc05f1b5c31981d8711130ac283355b7bfad403895f4096a6bb7e9a3d73a3bc
+# build failure with openssl 3
+CFLAGS+=" -Wno-error"
```

Comment:
import this patch instead https://github.com/taisei-project/taisei/pull/344

Reply:
done. that patch alone is not enough, as the released tarball does not include previous changes, even if they were made a year before its release. i added both